get-started/4-test-drive.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: page
+title: "Get Started: Test Drive an App"
```

Comment:
Not sure if we want "an App" there. You're test driving the tooling here, the SDK.

Reply:
Changed to just Test Drive